src/index.tsx

```diff
@@ -11,12 +11,10 @@ axios.defaults.baseURL = 'https://tindev-dev-deploy.herokuapp.com'
 const store = makeStore()
 
 ReactDOM.render(
-  <React.StrictMode>
```

Comment:
We should not remove `strictMode`

Reply:
MyBad. I removed it because the warning. Placing it back :smirk: .
However, kindly advise if you find a way to avoid the warning below:

<img width="1398" alt="Screenshot 2021-01-08 at 20 22 50" src="https://user-images.githubusercontent.com/42632188/104055795-a1e36500-51ef-11eb-8f1e-0e20e9306fcc.png">

Reply:
I think it's ok to ignore the warning for now, they won't affect the app as of current version of react